Demo/src/main/java/com/braintreepayments/demo/MainActivity.java

```diff
@@ -342,4 +341,36 @@ private void safelyCloseLoadingView() {
             mLoading.dismiss();
         }
     }
+
+    // NOTE: these are duplicated from PaymentMethodNonceInspector, which is package private
+    // we need getNonceDescription and getNonceTypeLabel for our integration test assertions,
```

Comment:
Shouldn't these be public so that merchants can access them? On iOS, they are available through `BTDropInResult`: https://github.com/braintree/braintree-ios-drop-in/blob/master/Sources/BraintreeDropIn/Public/BraintreeDropIn/BTDropInResult.h.